src/coreclr/vm/ceeload.h

```diff
@@ -915,6 +917,49 @@ class Module : public ModuleBase
         return (m_dwTransientFlags & IS_EDIT_AND_CONTINUE) != 0;
     }
 
+    BOOL IsInliningDisabledByProfiler() const
+    {
+        WRAPPER_NO_CONTRACT;
+        SUPPORTS_DAC;
+
+        return (m_dwTransientFlags & PROF_DISABLE_INLINING) != 0;
+    }
+
+    BOOL AreJITOptimizationsDisabled() const
+    {
+        WRAPPER_NO_CONTRACT;
+        SUPPORTS_DAC;
+
+#ifdef DEBUGGING_SUPPORTED
+        // check if debugger has disallowed JIT optimizations
+        //
+        // original CORDebuggerAllowJITOpts macro definition (we encapsulate it here to force all JIT optimization checks to cover both debugger and profiler bits):
+        // #define CORDebuggerAllowJITOpts(dwDebuggerBits)           \
+        //     (((dwDebuggerBits) & DACF_ALLOW_JIT_OPTS)             \
+        //      ||                                                   \
+        //      ((g_CORDebuggerControlFlags & DBCF_ALLOW_JIT_OPT) && \
+        //       !((dwDebuggerBits) & DACF_USER_OVERRIDE)))
```

Comment:
```suggestion
```
This comment is not useful. If somebody wants to dig into history, they can use source control.